lib/diagnostic.rb

```diff
@@ -74,7 +98,17 @@ def example_diff
   end
 
   describe 'PATCH update' do
-    # your test(s) here
+    before(:each) do
+      patch :update, id: Example.first.id, example: example_diff, format: :json
+    end
+
+    it 'is successful' do
+      expect(response).to be_success
+    end
+
+    it 'renders a JSON response' do
+      expect_json
```

Comment:
Whoops--I copied in my helper function and forgot to write it out. :\

     --Nick

On Mon, Apr 25, 2016 at 9:46 AM, Lauren Fazah <notifications@github.com>
wrote:

> In lib/diagnostic.rb
> <https://github.com/ga-wdi-boston/rails-api-bdd-diagnostic/pull/4#discussion_r60914769>
> :
>
> > @@ -74,7 +98,17 @@ def example_diff
> >    end
> >
> >    describe 'PATCH update' do
> > -    # your test(s) here
> > +    before(:each) do
> > +      patch :update, id: Example.first.id, example: example_diff, format: :json
> > +    end
> > +
> > +    it 'is successful' do
> > +      expect(response).to be_success
> > +    end
> > +
> > +    it 'renders a JSON response' do
> > +      expect_json
>
> Need to JSON.parse the response from the PATCH request with:
>
> expect(JSON.parse(response.body)).not_to be_nil
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/rails-api-bdd-diagnostic/pull/4/files/65bf668103cef500bfdc05724950cfeadf55dc53#r60914769>
>